### ISSUE TITLE
Couple of typos in twisted.internet.interfaces

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5593]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#5593|
|Type|defect|
|Created|2012-04-02 01:49:04Z|

"responsiblity" in `twisted.internet.interfaces.IHalfCloseableProtocol` and "continous" in `IPushProducer` are misspelled.

Attachments:

* [correctSpelling.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ce5/ce50b60c685ba31a1d359a5c68bce9c521fcded2/072283ff9d590e61cc1bf9c45e9d34c7c6508fce.patch) (1165 bytes) - added by chhabraamit on 2012-04-02 22:49:47Z - 
* [5593.misc](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ce5/ce50b60c685ba31a1d359a5c68bce9c521fcded2/375a65869cc9506ffd465ef449e19c29de4301e7.misc) (173 bytes) - added by chhabraamit on 2012-04-02 22:55:08Z - 
* [newCorrectPatch.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ce5/ce50b60c685ba31a1d359a5c68bce9c521fcded2/4d895ef7e15acdf2d0345fba415b927d962e7973.patch) (1566 bytes) - added by chhabraamit on 2012-04-06 00:08:23Z - New patch for the typo .
* [5593.2.misc](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ce5/ce50b60c685ba31a1d359a5c68bce9c521fcded2/56def06c451bac00de8ecaaab86f81f401a32b21.misc) (196 bytes) - added by chhabraamit on 2012-04-06 00:08:46Z - new corrected misc file.

<details><summary>Searchable metadata</summary>

```
trac-id__5593 5593
type__defect defect
reporter__itamar itamar
priority__lowest lowest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1333331344000000 1333331344000000
changetime__1333819538000000 1333819538000000
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="moijes12's avatar" src="https://avatars.githubusercontent.com/u/4566851?s=50" width="50" height="50">](https://github.com/moijes12)<a name="note_2"></a>|@moijes12 set owner to @moijes12|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="moijes12's avatar" src="https://avatars.githubusercontent.com/u/4566851?s=50" width="50" height="50">](https://github.com/moijes12)<a name="note_3"></a>|@moijes12 removed owner|
|-|-|

Thanks for the patch Amit. Below are my observations :

*  In the newsfile attachment 5593.misc
    *  The quotes should not applied around the name of the method/class for which the fix is made. For example, it should be twisted.internet.interfaces.IPushProducer and not "twisted.internet.interfaces.IPushProducer".
    *  The change from "responsiblity" to "responsibility" was made in twisted.internet.interfaces.IHalfCloseableProtocol.readConnectionLost and not in twisted.internet.interfaces.readConnectionLost.
    * I think the sentence could be rephrased a little ,to something like  "...in their respective docstrings.".
    * The file does not indicate where in the directory structure it fits into. Did you do a "svn add <filename>" ?
*  As part of the process, each patch must have a unit-test. But since this is for typos in the docstring, I'm not quite sure what to say. Probably the next reviewer could help with it.


Thanks once again.

**Comment by twisted-trac** [comment]
|<img alt="chhabraamit's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|chhabraamit commented|
|-|-|

Thank you moijes12 for your feedback.
New patch and misc file is uploaded.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for the patch and review.

Replying to [moijes12](#note_3):
> Thanks for the patch Amit. Below are my observations :

> *  In the newsfile attachment 5593.misc
A .misc file can be empty, as explained here: ReviewProcess#Newsfiles

I will apply the patch with an empty .misc file.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [34124]) Apply newCorrectPatch.patch: Correct a typo in the docstrings of IHalfCloseableProtocol.readConnectionLost and IPushProducer.

Author: chhabraamit
Reviewer: moijes12, thijs
Fixes: [#5593](https://github.com/twisted/twisted/issues/5593)
